src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py

```diff
@@ -11,12 +11,16 @@
 
 def _get_composite_data_sources_for_layup_provider(
     data_sources: CompositeDataSources, composite_definition_label: Optional[str] = None
-) -> DataSources:
+) -> Optional[DataSources]:
     """
     Prepare DataSources of composite for the DPF server.
```

Comment:
Should it be Prepare ``DataSources`` object of the composite for the DPF server?